### PR TITLE
Assignments 2,3,4 25/08/2022

### PR DESCRIPTION
**Pull request by emtec-jitendra-patil**
I have Submitted Assignments 2,3,4.

### CODE REVIEW

File: SQL Assignments 2,3,4.txt [hunk 1]
```diff
@@ -0,0 +1,458 @@
...
+--SELECT * FROM customers where snum = 1001;
+
+--Write a select command that produces the rating followed by the name of each customer in San
+--Jose
+
+--select rating,cname FROM customers where city='san jose';
+
+--Write a query that will give you all orders for more than Rs. 1,000
+
+--SELECT * FROM orders WHERE amt > 1000;
```
Comment (rishikesh-dhanawade-emtec) [inline SQL Assignments 2,3,4.txt]: Its good practice to use column names instead of *

Comment (emtec-jitendra-patil) [reply SQL Assignments 2,3,4.txt]: ok sir, got it

File: SQL Assignments 2,3,4.txt [hunk 2]
```diff
@@ -0,0 +1,458 @@
...
+
+--Write a query that will give you the names and cities of all salespeople in London with a
+--commission above 0.10
+
+--SELECT sname,city FROM salespeople WHERE comm > 0.10;
```
Comment (rishikesh-dhanawade-emtec) [inline SQL Assignments 2,3,4.txt]: Where is condition for City = London?

File: SQL Assignments 2,3,4.txt [hunk 3]
```diff
@@ -0,0 +1,458 @@
...
+
+--Write a query on the Customers table whose output will exclude all customers with a rating <=
+--100, unless they are located in Rome
+
+--SELECT cname,city = 'Rome',rating FROM customers WHERE rating <= 100;
```
Comment (rishikesh-dhanawade-emtec) [inline SQL Assignments 2,3,4.txt]: Need condition to check City = Rome

**Review by rishikesh-dhanawade-emtec — CHANGES_REQUESTED** [review]
check comments

**Comment by emtec-jitendra-patil** [comment]
Yes sir working on that

File: SQL Assignments 2,3,4.txt (hunk 1 above)
Comment (emtec-jitendra-patil) [reply SQL Assignments 2,3,4.txt]: done with changes

**Review by rishikesh-dhanawade-emtec — APPROVED** [review]
approved

**Pull request merged**